TombEngine/Objects/TR5/tr5_objects.cpp

```diff
@@ -114,11 +114,11 @@ static void StartEntity(ObjectInfo *obj)
 		obj->pivotLength = 50;
 		obj->radius = 102;
 		obj->intelligent = true;
-		obj->explodableMeshbits = 0x4000;
+		obj->HitRoutine = GuardHit;
 		obj->LotType = LotType::HumanPlusJump;
 		obj->SetBoneRotationFlags(6, ROT_X | ROT_Y);
 		obj->SetBoneRotationFlags(13, ROT_X | ROT_Y);
-		obj->SetupHitEffect();
+		obj->SetupHitEffect(true);
```

Comment:
Wrong use of SetupHitEffect().
Same for the other below except for attack_sub

Reply:
Take example from shiva if you need complex hitRoutine :)

Reply:
fixed